pkg/collector/corechecks/net/network/network.go

```diff
@@ -130,58 +153,72 @@ func (c *NetworkCheck) Run() error {
 	if err != nil {
 		return err
 	}
-	for _, interfaceIO := range ioByInterface {
-		if !c.isDeviceExcluded(interfaceIO.Name) {
-			submitInterfaceMetrics(sender, interfaceIO)
-		}
-	}
 
-	protocols := []string{"tcp", "udp"}
-	protocolsStats, err := c.net.ProtoCounters(protocols)
+	protocols := []string{"Tcp", "TcpExt", "Ip", "IpExt", "Udp"}
```

Comment:
Is the capitalization of the protocols correct? Previously we were using lower case protocols

Reply:
yeah this is alright, it is just a map key look up that [used to be lowercased](https://github.com/DataDog/datadog-agent/pull/33190/files#diff-56921c35d6d04a26faa0e166729301186c388b1cab4696322b41e3a25fbdd816L38) and is now [capitalized](https://github.com/DataDog/datadog-agent/pull/33190/files#diff-6d06c5238acf39860965b99ae992797800494c177969218ede4e62692b67ec94R299)